### PR TITLE
Added for articles page some new flex box styles and changed old

### PR DESCRIPTION
**Pull request by Sky7hunder**

### CODE REVIEW

File: app/assets/stylesheets/blog.scss [hunk 1]
```diff
@@ -102,36 +112,43 @@ nav {
...
 
-.nav-left, .nav-right {
-  display: inline-flex;
+.nav-right {
   align-items: center;
   width: 75%;
-  align-content: flex-end;
+  display: -webkit-box;
+  display: -moz-box;
+  display: -ms-flexbox;
```
Comment (galulex) [inline app/assets/stylesheets/blog.scss]: no prefixes

File: app/assets/stylesheets/blog.scss [hunk 2]
```diff
@@ -156,35 +173,61 @@ nav {
...
-      width: 49%;
-      float: left;
-      height: 41vh;
+//      display: inline-flex;
+//      position: absolute;
+//      top: 0;
+//      left: 0;
+//      right: auto;
+//      width: 49%;
+//      float: left;
```
Comment (galulex) [inline app/assets/stylesheets/blog.scss]: no comments

File: app/assets/stylesheets/blog.scss (hunk 1 above)
Comment (Sky7hunder) [inline app/assets/stylesheets/blog.scss]: ok

File: app/assets/stylesheets/blog.scss (hunk 2 above)
Comment (Sky7hunder) [inline app/assets/stylesheets/blog.scss]: I see. I left it because work in progress

File: app/assets/stylesheets/blog.scss [hunk 3]
```diff
@@ -156,35 +174,45 @@ nav {
...
-      position: absolute;
-      top: 0;
-      left: 0;
-      right: auto;
-      width: 49%;
-      float: left;
-      height: 41vh;
+      display: flex;
+      flex-direction: column;
+      -webkit-flex-flow: row wrap;
```
Comment (galulex) [inline app/assets/stylesheets/blog.scss]: do not use any prefixes. We use autoprefixer here

**Comment by galulex** [comment]
Attach screenshots please

**Comment by Sky7hunder** [comment]
fireshot capture 17 - active bridge - ruby on rails web development team - http___blog lvh me_3000_

**Comment by Sky7hunder** [comment]
In small window
fireshot capture 18 - active bridge - ruby on rails web development team - http___blog lvh me_3000_

**Comment by Sky7hunder** [comment]
fireshot capture 19 - active bridge - ruby on rails web developme_ - http___blog lvh me_3000_pikachu

File: app/assets/stylesheets/_disqus_thread.scss [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
...vRZl21MSFMJanZzTxTyUJEy2OJMqGWSfNOaxzrD+lpHlozhDbLz47exGBK0jhfeKNGCzcmai7hwh4hzKxdCosmXjvQ7/91zFkCF+RJtFFmmtMMqSKIXTSbPmz7tBYC/WuWnpgX31wtnoQwIoDW0GZx/U1HZ54tnwqhgUMfM9nWB6zPbcO2SThbUKsx/29tURmqv/zUFlVSQuyf2qLtiMy2qMKsmZqs1bhhf1WyMyOLq2zra7ts2L5Wi7CCzkm0jMNgC7gCyjJdX2OOtZBC1GzPvpZFUBd2ApMhss7ariHftXnnW5vMjN5Qz1w79V27tHA/Q/wsJbhfRbVNgKq1kADmRKH6S2tspdbopRCKCOrEU3P2ibun6xuy2+/S1Jc5xI7HLGllENVXEiyEpGbeabRhfglrbaMPNPo1xLpnu56am75U9LBqTl7W8CWSl0rkNMcKYYbYQp5GvJlc20PeNQcb/Vj/jyZ2F20DW3PAPLTlWY8YYmta86iiZjujkxiq2plmQ2Bp7WKtXsSw2qKxUdAjMAsje63T0SbRtefZZi3c6Nll15qIWp/sKVcPUb1W2y+GUGuCQnlD7mwYFFw2gc6yFwLESmq99Y3JT8Gq3WpIIWodBaRLVKmjtQy1nOR384ZJrBsxXp/RHHTOEt0ufX2uzTdLgR4Jtmy2CDPHLX40xNWRKvnZ660DV5K2WGsimpGMTkefjQvzWCOWJS5jmH/z2oG1lJSbt+Y2TzRpeTSwgxVtVs/xnrs378waf/aiJ+qGiZYCavZb+K3jyn2x/0xeLCqBvm9e2v25wCPlXQO8xZw5rBldWCNj/FYIB0TR7pykScwj5kquR2HO04faVu1Zn0Lw4BNJFpoXBauJXZt014I6G3RWcv8Farey6zmhsf0nqD2JKnM789IKuJ81eq3jcPai0tYFlaREymSPlLXRRqg6omHUPWaTGh+tJagJ/dDTYQmuULruqiZ2Ntu+067NWWdsP8r2Uc8F5/sKoUnUaFmELXhtY6R/34C2NU9z/UeQtXvHf/rzwOxKzj8PlOEeWcxGRf9k033mnwcEkKePzP9befg5K7MehLYyE8UW+PWQcp4BGuJd3xA/I2YhSDO4Rx8FuAfWkssuOh3ZugDC7NI3THbzjms+mhrf0/Fhqr2teQLgeU0BahOue/0Dq9BZJ7VWIsgAAAAASUVORK5CYII=);
```
Comment (galulex) [inline app/assets/stylesheets/_disqus_thread.scss]: sprockets has `asset_data_url` function or `assset_data_uri`

File: app/assets/stylesheets/blog.scss [hunk 5]
```diff
@@ -10,17 +11,26 @@ $lines-to-show: 4;
...
+}
+
 body {
   background: $dark-violet;
-  background: radial-gradient(#5F4076, #8A62AA);
+  background: radial-gradient(at top, #8A62AA, #5F4076);
```
Comment (galulex) [inline app/assets/stylesheets/blog.scss]: move all colors to variables

File: app/assets/stylesheets/blog.scss [hunk 6]
```diff
@@ -102,38 +128,46 @@ nav {
...
     align-items: center;
     align-self: center;
-    padding-right: 0;
-
-    &:after {
-      content: " | ";
-      padding: 0 5px;
-    }
+    padding-right: 15px;
+    border-right: 1px solid rgba(177, 177, 177, 0.5);
```
Comment (galulex) [inline app/assets/stylesheets/blog.scss]: move rgba colors to variables as well or use sass `rgba($violet, .3)`

**Comment by Sky7hunder** [comment]
fireshot capture 20 - active bridge - ruby on rails web development team - http___blog lvh me_3000_

**Pull request merged**